crates/pathfinder/src/state/sync/l2.rs

```diff
@@ -25,11 +25,32 @@ pub struct Timings {
     pub class_declaration: Duration,
 }
 
+/// A wrapper type to distinguish from the new global root in [sequencer::reply::Block]
+/// when sending [Event::Update].
+// FIXME
```

Comment:
Same as earlier with fixmes, these are not greppable.

Reply:
These FIXMEs are not present in the final state.